policies/cla.yml

```diff
@@ -0,0 +1,11 @@
+name: Contributor License Agreement Policy
+description: .NET Foundation CLA policy file
+
+resource: repository
+
+configuration:
+   cla:
+      content: https://raw.githubusercontent.com/dotnet-foundation/.github/main/CLA/dotnetfoundation.yml
+      minimalChangeRequired:
+         files: 1
+         codeLines: 1
```

Comment:
Intentionally not including the bot list. I'll explicitly update that list as needed for the repo to keep it minimal and within the actual scope of automated PRs expected/received